static/js/stream_popover.js

```diff
@@ -262,6 +275,26 @@ function build_starred_messages_popover(e) {
 
 }
 
+function build_move_topic_to_stream_popover(e, message_id, stream_id) {
+    const elt = e.target;
+    const streams = stream_data.subscribed_subs().filter(s => s.stream_id !== stream_id);
+    const content = render_move_topic_to_stream({
+        message_id,
+        streams,
+    });
+
+    $(elt).popover({
+        content: content,
+        html: true,
+        trigger: "manual",
+        fixed: true,
+    });
```

Comment:
The positioning seems like it can be a little weird :/

![image](https://user-images.githubusercontent.com/2746074/75938325-55473480-5e3c-11ea-8169-fa4411a80fdc.png)

Reply:
I see this on my local machine, what browser are you using? 
![image](https://user-images.githubusercontent.com/3765508/75941476-d236d600-5e5d-11ea-8f0e-b17d4703396a.png)

Reply:
Ohhh I got it!! let me fix it, thanks!